isis/src/control/objs/BundleUtilities/BundleObservationVector.cpp

```diff
@@ -148,16 +167,28 @@ namespace Isis {
         solveSettings = bundleSettings->observationSolveSettings(observationNumber);
       }
 
-      isisObservation->setSolveSettings(solveSettings);
-
-      bundleObservation.reset(isisObservation);
+      if (isis) {
+        isisObservation->setSolveSettings(solveSettings);
+        bundleObservation.reset(isisObservation);
+      }
+      else {
+//        csmObservation->setSolveSettings(solveSettings); doesn't exist
```

Comment:
This does exist, see CsmBundleObservation::setSolveSettings

Reply:
Thanks @jessemapel forgot to update that. It didn't exist, I think, before the merge.